openedx/core/djangoapps/user_authn/views/tests/test_register.py

```diff
@@ -394,6 +395,10 @@ class RegistrationViewTest(ThirdPartyAuthTestMixin, UserAPITestCase):
     def setUp(self):  # pylint: disable=arguments-differ
         super(RegistrationViewTest, self).setUp()
         self.url = reverse("user_api_registration")
+        patcher = mock.patch(
```

Comment:
I guess we should revert it now.

Reply:
No, As we cannot revert form imports we need this change as well.